lib/presentation/ui/home/home_page.dart

```diff
@@ -181,7 +182,7 @@ class RecentListWidget extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return StreamBuilder<List<ParishModel>>(
+    return StreamBuilder<List<OccurrenceModel>>(
```

Comment:
What is the snapshot hasn't a data? There's not `initialValue` provided, thus accessing `data` without in a non null-aware way may not be safe.

Reply:
This was not thought for the final version, it was just a way for me to test each endpoint to see if all was well.

Do you want me to remove it?